### ISSUE TITLE
fastqc_report without switch button for plots by Captus and FastQC

### ISSUE THREAD

**Issue by bmelchorcastro** [edgardomortiz/Captus#16]
Hi, Edgardo!

First and foremost, congratulations on this excellent toolkit!

I hope you can assist me with this matter. I received the fastac_report after the cleaning step, but I did not see the option to switch tabs and compare the plot produced by Captus to the corresponding plot from FastQC. 

First, I executed it as you suggested in the pipeline:
`captus_assembly clean --reads ./00_raw_reads`

But, as you mentioned in the tutorial, falco is the default argument, so I ran it as follows:
`captus_assembly clean -r ./00_raw_reads  --qc_program fastqc`

Even after that, I did not see the fastqc tab button.
I have attached the file so you can see it.
[captus-assembly_clean.report3.zip](https://github.com/user-attachments/files/17097018/captus-assembly_clean.report3.zip)

Thank you in advance for your time, and I hope you can help me.

**Comment by edgardomortiz** [comment]
Hi @bmelchorcastro ,

I think by button you must be referring to the tabs in our tutorial where we show a direct comparison between Captus' and FastQC's graphs (https://edgardomortiz.github.io/captus.docs/assembly/clean/report/index.html). This was done for illustrative purposes only, so the user can understand the direct equivalence of our graphs to FastQC's (which use the data produced by FastQC).

In short, the cleaning report does not show the FastQC graphs, if you are still interested in those they are kept inside the subdirectories of the cleaning directory you chose.

Edgardo
